x-pack/plugins/ml/public/application/components/import_export_jobs/export_jobs_flyout/jobs_export_service.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// @ts-expect-error
+import { saveAs } from '@elastic/filesaver';
+import type { MlApiServices } from '../../../services/ml_api_service';
+
+export class JobsExportService {
+  constructor(private _mlApiServices: MlApiServices) {}
+
+  public async exportAnomalyDetectionJobs(jobIds: string[]) {
+    const configs = await Promise.all(jobIds.map(this._mlApiServices.jobs.jobForCloning));
+    const configsForExport = configs.length === 1 ? configs[0] : configs;
+    const blob = new Blob([JSON.stringify(configsForExport, null, 2)], {
+      type: 'application/json',
+    });
+    saveAs(blob, 'ml_jobs.json');
```

Comment:
**nit:** I think it might be convenient to have jobs type, stack version, and date in the file name, something like `anomaly_detection_ml_jobs_7.14.0_20210808`

Reply:
Yeah this is a good idea to give these files a more specific name.
I've made a change here https://github.com/elastic/kibana/pull/101037/commits/330129a6c019ee61c6ac51bc5bdc067c930b711b so that the files are now either  `anomaly_detection_jobs.json` or `data_frame_analytics_jobs.json`.

There are problems with adding the version to the file name as it may be coming from a mixed version cluster.
For now I think these names are definitely an improvement and we can make changes later on if we decide we should add more information.